Source/WebCore/crypto/mac/CryptoAlgorithmAES_GCMMac.cpp

```diff
@@ -43,12 +44,20 @@ ALLOW_DEPRECATED_DECLARATIONS_BEGIN
 ALLOW_DEPRECATED_DECLARATIONS_END
     if (status)
         return Exception { ExceptionCode::OperationError };
-
     memcpy(cipherText.data() + plainText.size(), tag.data(), desiredTagLengthInBytes);
 
     return WTFMove(cipherText);
 }
 
+static ExceptionOr<Vector<uint8_t>> encryptCryptoKitAES_GCM(const Vector<uint8_t>& iv, const Vector<uint8_t>& key, const Vector<uint8_t>& plainText, const Vector<uint8_t>& additionalData, size_t desiredTagLengthInBytes)
+{
+    Vector<uint8_t> cipherText(plainText.size() + 16);
```

Comment:
Let's give 16 a name so we don't have any magic numbers.  It looks like some kind of header size.

Reply:
done.